src/utils/apollo-client.ts

```diff
@@ -0,0 +1,11 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { RestLink } from 'apollo-link-rest';
+
+const restLink = new RestLink({ uri: process.env.STRAPI_URL });
+
+const client = new ApolloClient({
```

Comment:
Can we call this strapiRestClient ?

Reply:
Much better if we can create ClientFactory, which will give us the respective type of ApolloClient - Lambda Client, Strapi Client etc. Just a thought, this works for now but it we have more types, factory pattern works better.

Reply:
Ah I like the factory idea, I think that will resolve the issues that I was having with the tests as well.